content/_data/changelogs/lts.yml

```
@@ -4785,6 +4785,327 @@
+      - url: "https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.32.v20200930"
+        title: Jetty 9.4.32.v20200930 changelog
+      - url: "https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.33.v20201020"
+        title: Jetty 9.4.33.v20201020 changelog
+      - url: "https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.34.v20201102"
+        title: Jetty 9.4.34 changelog
+      - url: "https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.35.v20201120"
+        title: Jetty 9.4.35 changelog
+    message: |-
+      Winstone 5.13: Update Jetty from 9.4.30.v20200611 to 9.4.35.v20201120 for bug fixes and enhancements.
+
+  - type: rfe
+    category: rfe
+    pull: 4962
+    authors:
+    - StefanSpieker
+    message: |-
+      Drop support for deprecated system properties: <code>hudson.model.Hudson.logStartupPerformance</code>, <code>hudson.model.Hudson.initLogLevel</code>, <code>hudson.model.Hudson.parallelLoad</code>, <code>hudson.model.Hudson.killAfterLoad</code> and <code>hudson.model.Hudson.workspaceDirName</code>.
+      Please use <code>jenkins.model.Jenkins.</code>-prefixed SystemProperties.
+
+  - type: rfe
+    category: rfe
+    pull: 5063
+    authors:
+    - fqueiruga
+    - Wadeck
+    message: |-
+      Reduce page load time by loading the administrative monitors popup on demand.
+      Allow keyboard navigation even when there are active administrative monitors.
+
+  - type: rfe
+    category: rfe
+    pull: 5111
+    authors:
+    - timja
+    references:
+      - pull: 5111
+      - url: "https://github.com/stapler/stapler/releases/tag/stapler-parent-1.262"
+        title: Stapler 1.262 changelog
+    message: |-
+      Update stapler to 1.262 to fix a number of IllegalReflectiveAccessWarnings when running on Java 11.
+
+  - type: rfe
+    category: rfe
+    pull: 5082
+    issue: 58101
+    authors:
+    - res0nance
+    message: |-
+      Reduce lock contention around jenkins queue.
+
+  - type: rfe
+    category: rfe
+    pull: 5129
+    authors:
+    - dependabot[bot]
+    references:
+      - pull: 5129
+      - url: "https://github.com/jnr/jnr-posix/compare/jnr-posix-3.0.45...jnr-posix-3.1.4"
+        title: Commits from jnr-posix 3.0.45 to 3.1.4
+    message: |-
+      Update jnr-posix library from 3.0.45 to 3.1.4.
+
+  - type: rfe
+    category: rfe
+    pull: 5125
+    authors:
+    - dependabot[bot]
+    references:
+      - pull: 5125
+      - url: "https://github.com/java-native-access/jna/blob/master/CHANGES.md#release-560"
+        title: JNA 5.6.0 changelog
+      - url: "https://github.com/java-native-access/jna/blob/master/CHANGES.md#release-550"
+        title: JNA 5.5.0 changelog
+      - url: "https://github.com/java-native-access/jna/blob/master/CHANGES.md#release-540"
+        title: JNA 5.4.0 changelog
+    message: |-
+      Update Java native access (jna) library from 5.3.1 to 5.6.0 for most recent platform library fixes and enhancements.
```

most is incorrect as 5.7.0 is out (which is required for full mac m1 support)
```suggestion
      Update Java native access (jna) library from 5.3.1 to 5.6.0 for more recent platform library fixes and enhancements.
```